.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: java
 
 jdk:
-  - oraclejdk8
+  - openjdk13
```

Comment:
Shouldn't this be an addition instead of a replacement ?
I mean we don't want it to work with Java 8 anymore ?